apps/server/src/modules/tldraw/redis/tldraw-redis.factory.ts

```diff
@@ -0,0 +1,31 @@
+import { Redis } from 'ioredis';
```

Comment:
Why do you use the ioredis package and not the redis package?

Reply:
When we started development we noticed that redis package was terribly outdated and we didn't want to update it because it was used in other parts of the application. Seems to have been a correct choice since I've seen Max reverting his recent redis package update due to some problems.

So we chose ioredis, which is even more popular than redis, and this way we don't have to worry it will break anything in the existing code.

Reply:
It's right that there where an issue at first try of update the redis package, but issue where solved.
Maybe in wone of the next iterations this change to own package for all.
To reduce the amount of redis package who must be update.